dist.ini

```diff
@@ -196,6 +195,7 @@ ExtUtils::MakeMaker = 6.64
 
 [Prereqs / ConfigureRequires]
 -phase = configure
+parent              = 0
```

Comment:
I'm not sure this is required at configure time for this module?

Reply:
Ah, good catch. I'll change that